src/__tests__/events.js

```diff
@@ -137,6 +137,90 @@ const eventTypes = [
   },
 ]
 
+const bubblingEvents = [
```

Comment:
I think it would be simpler if this were split into two arrays and they were arrays of strings only. e.g. `bubblingEvents` and `nonBubblingEvents`.

Reply:
This was actually how I originally wrote it :)
Done!